api/src/main/java/org/openmrs/PersonAttributeType.java

```diff
@@ -14,12 +14,13 @@
 
 import org.codehaus.jackson.annotate.JsonIgnore;
 import org.hibernate.search.annotations.Field;
+import org.openmrs.attribute.BaseAttributeType;
 import org.openmrs.util.OpenmrsUtil;
 
 /**
  * PersonAttributeType
  */
-public class PersonAttributeType extends BaseChangeableOpenmrsMetadata implements java.io.Serializable, Comparable<PersonAttributeType> {
+public class PersonAttributeType extends BaseAttributeType<Person> implements Comparable<PersonAttributeType> {
```

Comment:
@ibacher  I couldn't implement `public class PersonAttributeType extends BaseAttributeType<Person> implements Comparable<PersonAttributeType> {` because of this error `the interface comparable cannot be implemented more than once with different arguments: comparable<attribute> and comparable<personattribute>` that is why I have `public class PersonAttribute extends BaseAttribute<PersonAttributeType, Person> implements org.openmrs.attribute.Attribute<PersonAttributeType, Person> {` instate

Reply:
Ok... the main purpose of this suggestion was to add the `Serializable` interface.